### ISSUE TITLE
Hardcoded configuration

### ISSUE THREAD

**Issue by jgonggrijp** [spamgourmet/spamgourmet#40]
There are some places in the code where filesystem paths appear hardcoded in the source code. In particular, I ran into these two in the spameater script:

https://github.com/spamgourmet/spamgourmet/blob/5d57e036d202c0c9322119e39c3cefa8b7fe9d3f/mailhandler/spameater#L22

https://github.com/spamgourmet/spamgourmet/blob/5d57e036d202c0c9322119e39c3cefa8b7fe9d3f/mailhandler/spameater#L32

There are probably other places and possibly other types of configuration that shouldn't be hardcoded. Getting these factored out to something that is not committed to VCS would greatly help in reproducibility. See for example also #6.

**Comment by mikedlr** [comment]
Those are substitution targets - effectively macros. During installation with `spamgourmet-clone` they get fixed - see

https://github.com/spamgourmet/spamgourmet/blob/master/spamgourmet-clone/scripts/install-spamgourmet.sh

that's a kind of ugly way do things but it does work and means they are only partly hardcoded in those locations.  Maybe this ticket should be retitled "tidy up installation process" with some ideas for how to make this all neater?

**Comment by jgonggrijp** [comment]
@mikedlr Makes sense. I don't have any ideas, because I'm unfamiliar with best practices in the Perl-CGI world.
